src/segmentmytiff/utils/datasets.py

```diff
@@ -19,7 +20,7 @@ def image_path_to_mask_path(image_path: Path) -> Path:
         self.masks = [str(image_path_to_mask_path(Path(p))) for p in self.images][:self.limit]
         non_existing_masks = [p for p in self.masks if Path(p).exists() == False]
         if non_existing_masks:
-            print(f"{len(non_existing_masks)} of a total of {len(self.masks)} masks not found.")
+            logging.getLogger().warning(f"{len(non_existing_masks)} of a total of {len(self.masks)} masks not found.")
```

Comment:
depends on how we would like this function envolve, maybe consider initiate a `logger` to collect all the logs? Like you did in other modules.

Reply:
good catch! thanks!